Common/GameSystem/CardRegistry.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria.ModLoader;
+using TerraTCG.Common.GameSystem.CardData;
+using TerraTCG.Common.GameSystem.GameState;
+using TerraTCG.Common.GameSystem.PackOpening;
+
+namespace TerraTCG.Common.GameSystem
+{
+
+	public class CardRegistry : ModSystem
+	{
+		public static CardRegistry Instance;
+
+		public static Action CardChanges;
```

Comment:
https://github.com/Team-eva-and-Don/TerraTCG/blob/566c0a2342bb21040edf78245e13365ecca24995/Common/UI/Common/UIBetterTextbox.cs#L33-L39

この辺を参考にすると
- event修飾子をつけると良い
- `OnCardsAdded`の名前のほうが良い（関数に合わせて）


```suggestion
		public static event Action OnCardsAdded;
```